### PR TITLE
update postgres, add starter source code

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

File: src/main/java/org/example/model/Cat.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        return owners;
+    }
+
+    public void addOwner(Owner owner) {
+        owners.add(owner);
+    }
+
+    @Override
+    public String toString() {
+        return "Cat{" +
```
Comment (developerjaym) [inline src/main/java/org/example/model/Cat.java:61]: You may have done this elsewhere, but consider warning students about generated toStrings.
Like if this Cat class's toString printed out the toStrings of the Owners and Owner printed out the toStrings of Cats.
I think I remember some of the BlackRock students dealing with circular toString issues.

**Review by developerjaym — APPROVED** [review]

File: src/main/java/org/example/model/Cat.java (hunk 1 above)
Comment (linda-seiter) [reply src/main/java/org/example/model/Cat.java:61]: I added a warning about this in the lab, I'll update the lesson too.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,122 @@
-# JPA Relationship Lab 3
+# JPA Relationship Lab 2
```
Comment (kcleland0818) [inline README.md]: Only here to serve as a reminder to maybe rename the repo on GitHub once merged so this doesn't become confusing later on to students and instructors.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,122 @@
...
+| 2   | Yuri |
+| 3   | Luka |
+
+`SELECT * FROM CAT;`
+
+| ID  | AGE | BREED              | NAME            |
+|-----|-----|--------------------|-----------------|
+| 4   | 5   | Tabby              | Meowie          |
+| 5   | 1   | Birman             | Whiskers        |
+| 6   | 10  | American Shorthair | Freddy Purrcury |
```
Comment (kcleland0818) [inline README.md:96]: LOL "Freddy Purrcury". Going to name my next male cat that

**Review by kcleland0818 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (linda-seiter) [reply README.md:96]: :-)

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md]: fixed, thanks.

**Review by developerjaym — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**